src/vm/tests/test-update.js

```diff
@@ -241,6 +241,18 @@ var PAYLOADS = {
         max_shm_memory: 1234
     }, remove_cpu_cap: {
         cpu_cap: 0
+    }, add_fs_tmp_global: {
+        add_filesystems: [
+            {
+                type: 'lofs',
+                source: '/tmp',
+                target: '/var/tmp/global',
+                options: ['nodevice']
+            }
+        ]
+    }, remove_fs_tmp_global: {
+        remove_filesystems: [
+	  '/var/tmp/global']
     }
 };
```

Comment:
is there a specific reason we don't have an "update_filesystems" test(s)?

Reply:
Mostly because I only changed remove_filesystems, so I wanted to add tests for that, but needed add_filesystems to do so. I could look into some update_filesystems tests too, maybe update the options and see if they stick.

Reply:
Blah, looks like update_filesystems is also not implemented!

I'll see if it is easy to implement, if not I will remove the reference to it in the manpage

Reply:
Could you document that update isn't supported rather than removing all mention, thanks. Same for pci_devices I guess - honestly I forget what we did there

Reply:
There is some weird magic happening with expander.js and proptable.js... I am going to see if I can implement it first, I'll temporarily push an extra commit so I can drop it if it doesn't work out. (Given that I build form this branch for testing)

But it looks like for pci_devices it does a remove then add if a property changes, which makes sense as there seems to be no code to select a device then add/remove a sub prop.